staff/joel-molero/cv/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum Vitae</title>
+</head>
+<body>
+    <header>
+        <h1>Joel Molero</h1>
+        <br>
+        <address>Address: Spain, Barcelona, Febot street, Nº24, 2-1 
+            <br>
+            Tel: 93 710 63 28 || Mobile: 601 72 34 21
+            <br>
+            Email: joel.molero@gmail.com
+        </address>
+    <hr>
+    </header>
+    <main>
+        <br>
+        <article>
```

Comment:
instead of article, section applies better here. to understand the meanings of each, check the following links:
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section